src/Microsoft.AspNet.Http.Abstractions/Extensions/UseMiddlewareExtensions.cs

```diff
@@ -166,7 +192,12 @@ private static object GetService(IServiceProvider sp, Type type, Type middleware
             var service = sp.GetService(type);
             if (service == null)
             {
-                throw new InvalidOperationException(Resources.FormatException_InvokeMiddlewareNoService(type, middleware));
+                throw new InvalidOperationException(
+                    string.Format(
+                        Resources.Exception_InvokeMiddlewareNoService,
+                        type,
+                        middleware
+                    ));
```

Comment:
I assume this was because the resource API changes changed? That said, don't use this format here, it's out of place.
`throw new InvalidOperationException(string.Format(esources.Exception_UseMiddlewareNoInvokeMethod, InvokeMethodName));`